src/Physics/babylon.physicsEngine.ts

```diff
@@ -54,6 +56,10 @@
             this.gravity = gravity || new Vector3(0, -9.807, 0);
             this._currentPlugin.setGravity(this.gravity);
         }
+		
+		public _getGravity() : Vector3 {
```

Comment:
the rest of the functions (like setGravity) also have an underscrore, i assumed it is due to the direct interaction with the plugin. Can be taken out actually (for all other functions as well)